lib/screens/login_page.dart

```diff
@@ -105,116 +85,272 @@ class _LoginPageState extends State<LoginPage> {
     }
   }
 
-  Widget _form(BuildContext context) {
-    return Form(
+  Widget _loginTextForm(context) {
+    return Center(
+      child: Form(
         key: _formKey,
```

Comment:
It seems like the loading spinner is broken for when its waiting to be logged back in. Can you look for a way to add a spinner back?